test/config.flat

```diff
@@ -1,3 +1,10 @@
 port: 1234
 host: https://ao.computer
-await-inprogress: false
\ No newline at end of file
+await-inprogress: false
+routes/1/template: /result/.*
+routes/1/node/prefix: http://cu:6363
+routes/2/template = /graphql
+routes/2/nodes/1/prefix = https://arweave-search.goldsky.com
+routes/2/nodes/1/opts/http_client/httpc
+routes/2/nodes/2/prefix = https://arweave.net
+routes/2/nodes/2/opts/http_client = gun
```

Comment:
The docs say that these complex types won't work. Has the config.flat file format been improved?

https://github.com/permaweb/HyperBEAM/blob/main/docs/misc/installation-core/hyperbeam-setup-config/configuration/configuration-methods.md?plain=1#L34-L36

Even if so, I think these need to be `:` colons, not `=` equals signs.

Reply:
https://github.com/permaweb/HyperBEAM/pull/163#discussion_r2084859893